### DISCUSSION TITLE
Is there a way to query the number of rollbacks within an organization?

### DISCUSSION THREAD

**Discussion by alaska907486** [community/community#54186] [Programming Help]
### Select Topic Area

Question

### Body

Is there a way to query the number of rollbacks within an organization?  Our organization is looking at improving this statistic but first need the ability to query the number that have taken place to create a baseline and then continue to track against it to see improvement.

Thank you for your help.

**Comment by joniadaniel** [comment]
You can see that the syntax of the rollback SQL statement is simple. You just have to write the statement ROLLBACK TRANSACTION, followed by the name of the transaction that you want to rollback.
regards: [best camping fan for large tent](https://campingstartup.com/best-camping-fans-for-tents/)

**Reply by alaska907486** [reply]
Hi Jonia thank you for replying.... I understand the process of rolling back.  I want to query the number of times rolled back during a certain period and haven't found any guidance so was wondering if others found a way or if its just not available.  Thank you.

**Comment by ghost** [comment]
Hola si es posible consultar el número de reversiones dentro de una organización si se dispone de acceso a los registros y herramientas adecuadas. 

Aqui hay algunas formas para hacerlo:

Sistema de control de versiones: Si la organización utiliza un sistema de control de versiones, como Git, Subversion o Mercurial, puedes utilizar comandos o herramientas específicas para obtener información sobre las reversiones. Por ejemplo, con Git puedes utilizar el comando git log para ver el historial de commits y contar el número de reversiones realizadas.

Herramientas de seguimiento de problemas: Muchas organizaciones utilizan herramientas de seguimiento de problemas como JIRA, Bugzilla o Trello para realizar un seguimiento de los problemas y cambios en el software. Estas herramientas suelen registrar los cambios realizados en forma de tickets o incidencias. Puedes buscar en estas herramientas los tickets o incidencias marcados como "reversión" o con etiquetas relacionadas y contar su número.

Registros de cambios o informes de versiones: Algunas organizaciones mantienen registros de cambios o informes de versiones donde se documenta el historial de cambios realizados en el software. Estos registros pueden incluir información sobre las reversiones realizadas. Puedes revisar estos registros o informes para contar el número de reversiones.

Consultar al equipo de desarrollo: Si no tienes acceso directo a los registros o herramientas mencionadas anteriormente, puedes preguntar al equipo de desarrollo de la organización.
